src/main/external-resources/renderers/Sony-PlayStation3.conf

```diff
@@ -39,14 +39,15 @@ Supported = f:mp4        v:mp4|h264               a:aac                m:video/m
 Supported = f:wmv        v:wmv|vc1                a:wma                m:video/x-ms-wmv           n:2
 
 # Supported audio formats:
+Supported = f:aac     a:(?!alac).+   n:2             m:audio/x-m4a
+Supported = f:atrac                  n:2             m:audio/x-oma
```

Comment:
Not correct.
https://tools.ietf.org/html/rfc5584
audio/ATRAC3, audio/ATRAC-X, and  audio/ATRAC-ADVANCED-LOSSLESS

I just discovered some extensions used for ATRAC files are missing:  https://en.wikipedia.org/wiki/Adaptive_Transform_Acoustic_Coding
.aa3
.oma
.at3
.at9

If i believe the FFMpeg code they are suppoted (not sure about .at9 though...):
https://www.ffmpeg.org/doxygen/3.2/atrac3_8c_source.html